src/metabase/search/impl.clj

```diff
@@ -183,13 +180,9 @@
                                   (when collection_effective_ancestors
                                     {:effective_ancestors collection_effective_ancestors})))
          :scores          (remove-thunks all-scores))
-        (update :dataset_query (fn [dataset-query]
-                                 (when-let [query (some-> dataset-query json/decode)]
-                                   (if (get query "type")
-                                     (mbql.normalize/normalize query)
-                                     (not-empty (lib/normalize query))))))
         (dissoc
          :all-scores
+         :dataset_query
```

Comment:
This field was never used by the FE, but did magically add extra processing by the MLv2 library. Since New Search does not return this, the null value would cause that processing to crash the entire app, so we have removed it.

It would be nice if the FE was more resilient in this regard, or at least gave a user friendly error...